build.gradle

```diff
@@ -4,10 +4,9 @@ buildscript {
     repositories {
         maven { url 'https://maven.google.com' }
         jcenter()
-        google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.0'
+        classpath 'com.android.tools.build:gradle:3.1.0'
```

Comment:
why do we need to downgrade the gradle tool and  gradle version? it should be a way to use latest version right? Gradle 3 is quite old already. I thought we are updating to the latest version?

Reply:
Yes but have you try publishing artifact to Bintray? Previously when use latest Gradle version @askur87 can't release to Bintray, he told me so I just downgrade it as a fastest solution and to minimise the risk since this repository actually use older Gradle version. 
I think payment team need to spike it first to find pros, cons, best approach, etc, and write it as an ADR since publishing script seems like not compatible with latest Gradle version and the script need to be update.

Reply:
ok. I pushed a commit to remove a redundant field in gradle that was needed for Gradle 4. it will be good to go now

Reply:
Thanks @theanh0512, I'll resolve this thread